.github/workflows/ci.yml

```diff
@@ -0,0 +1,78 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - '**'
```

Comment:
Seems you can omit this since all branches is the default https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#push

Reply:
Tried without the branches section on another repo, and this no longer triggered builds on push.
I suspect using this long form syntax overrides the default on push.
We want to keep this syntax to control builds on tags (in relation to version management with reckon & tags).

Reply:
Ok, thanks for checking.